src/main/java/nextstep/optional/User.java

```diff
@@ -33,7 +35,7 @@ public static boolean ageIsInRange1(User user) {
     }
 
     public static boolean ageIsInRange2(User user) {
-        return false;
+        return Optional.ofNullable(user).filter(x -> x.getAge() != null && x.getAge() >= 30 && x.getAge() <= 45).isPresent();
```

Comment:
```suggestion
        return Optional.ofNullable(user)
          .filter(x -> x.getAge() != null && x.getAge() >= 30 && x.getAge() <= 45)
          .isPresent();
```

한줄에 하나씩 나누는게 가독성과 디버깅의 장점이 있더라구요 😄

Reply:
```suggestion
        return Optional.ofNullable(user).filter(x -> x.getAge() != null && validRange()).isPresent();
```

함수로 분리시켜도 가독성에 좋겠네요 ! 😄